channels/mx.m3u

```diff
@@ -59,8 +51,6 @@ http://d3kr0d4mfjxpbv.cloudfront.net/p/SPAS_1920x1080_4000k.m3u8
 https://d2416wdkjdz9sf.cloudfront.net/forotv/forotvm.m3u8?fluxustv.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/hMtKE60.png" group-title="",Genesis TV
 http://201.144.184.98:1935/genesis/smil:television.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/CA0LESg.jpg" group-title="",Ingenio TV
-https://srvstream01.televisioneducativa.gob.mx/hls/ingeniotv.m3u8
```

Comment:
Link works (Tested in VLC)

Reply:
Still not working. VLC output:
<img width="740" alt="Screenshot 2019-10-12 at 20 17 20" src="https://user-images.githubusercontent.com/7253922/66705292-582a7700-ed2d-11e9-9e9c-c360ca21f61b.png">

Reply:
Still works for me